### PR TITLE
Install functions from `babel.fish`

### PR DESCRIPTION
**Pull request by 0risc**
Fixes https://github.com/users/Neved4/projects/18/views/1?pane=issue&itemId=87547381

### CODE REVIEW

**Review by Neved4 — CHANGES_REQUESTED** [review]
Add some concise note after installation to guide users how to use `babel.fish`.

**Comment by 0risc** [comment]
Done!

File: Formula/babelfish.rb [hunk 1]
```diff
@@ -27,6 +27,22 @@ def install
...
+        $ source /opt/homebrew/share/fish/vendor_functions.d/babel.fish
+
+      This allows you to run a bash script like this:
+
+        $ source chruby.sh
+        $ chruby
+           ruby-2.5
+           ruby-2.6
+           ruby-2.7
+       CAVEATS
```
Comment (Neved4) [inline Formula/babelfish.rb]: Fix `brew style`'s alignment:
```suggestion
    CAVEATS
```

**Review by Neved4 — CHANGES_REQUESTED** [review]
Minor fix.

File: Formula/babelfish.rb (hunk 1 above)
Comment (0risc) [reply Formula/babelfish.rb]: this was already fixed in 255e9f810bb9a56487926edc89963889e9d91d10

File: Formula/babelfish.rb [hunk 2]
```diff
@@ -24,6 +24,23 @@ def install
         gcflags: "all=-l -B -wb=false",
       )
     doc.install "README.md"
+    fish_function.install "babel.fish"
+  end
+
+  def caveats
+    <<~CAVEATS
+      The shell hook has been installed to which you can use by running:
```
Comment (Neved4) [inline Formula/babelfish.rb:32]:
```suggestion
      The shell hook has been installed, you can use it by running:
```

File: Formula/babelfish.rb [hunk 3]
```diff
@@ -24,6 +24,23 @@ def install
...
+
+        $ source /opt/homebrew/share/fish/vendor_functions.d/babel.fish
+
+      This allows you to run a bash script like this:
```
Comment (Neved4) [inline Formula/babelfish.rb:36]:
```suggestion
      Then you can run shell scripts like this:
```

File: Formula/babelfish.rb (hunk 2 above)
Comment (0risc) [reply Formula/babelfish.rb]: Thx apparently I was pretty tired when I wrote this haha

**Pull request closed without merging**